app/aboutUs.html

```diff
@@ -130,4 +135,7 @@ <h3><strong>Benefits:</h3></strong>
       <div class="gap"></div>
```

Comment:
line 128, button not working at the moment. Refer to header.html dealing with onclick event with membership button in your local branch.

Reply:
Same with line 97